src/tests/__snapshots__/App.test.test.js.snap

```diff
@@ -1,75 +1,72 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Testing App component Snapshot testing 1`] = `
-Array [
+<div
+  className="app"
+>
   <div
-    className="app"
+    className="container"
   >
-    <div
-      className="container"
-    >
-      <div>
+    <div>
+      <div
+        className="header"
+      >
         <div
-          className="header"
+          className="search-bar"
         >
-          <div
-            className="search-bar"
+          <form
+            onSubmit={[Function]}
           >
-            <form
-              onSubmit={[Function]}
+            <input
+              onChange={[Function]}
+              placeholder="search for movies"
+              type="text"
+              value=""
+            />
+            <button
+              type="submit"
             >
-              <input
-                onChange={[Function]}
-                placeholder="search for movies"
-                type="text"
-                value=""
+              <i
+                className="fa fa-search"
               />
-              <button
-                type="submit"
-              >
-                <i
-                  className="fa fa-search"
-                />
-              </button>
-            </form>
-          </div>
+            </button>
+          </form>
         </div>
+      </div>
+      <div
+        className="banner-img"
+      />
+      <div
+        className="movie-wrapper"
+      >
         <div
-          className="banner-img"
-        />
-        <div
-          className="movie-wrapper"
+          className="movie-list"
         >
           <div
-            className="movie-list"
+            className="movie-container"
           >
             <div
-              className="movie-container"
+              className="movies-error"
             >
-              <div
-                className="movies-error"
-              >
-                <h3 />
-              </div>
+              <h3 />
             </div>
           </div>
+        </div>
+        <div
+          className="show-list"
+        >
           <div
-            className="show-list"
+            className="movie-container"
           >
             <div
-              className="movie-container"
+              className="shows-error"
             >
-              <div
-                className="shows-error"
-              >
-                <h3 />
-              </div>
+              <h3 />
             </div>
           </div>
         </div>
       </div>
     </div>
-  </div>,
-  ",",
-]
```

Comment:
As requested by the previous reviewer kindly ensure that you follow the design guidelines, including the colors (select one main color for your website as below), typography (font face, size and weight) and layout (composition, and space between elements)
Your images look bigger than the actual design and also missing the arrow with a round circle on the top right corner. For instance, the image should be smaller on the top middle with the arrow in round circle on the top right, and movie title and year below them ( just as the country and number is below the flag in the actual design). Kindly check and fix all these issues in order to match the actual design. :pray: 


| Your home design | Actual design|
| --- | --- |
![Screenshot from 2023-01-13 20-13-57](https://user-images.githubusercontent.com/65233860/212411027-61adaec4-ed8a-43e9-94e5-e5b84a25dd75.png)  |  ![Screenshot from 2023-01-13 20-13-31](https://user-images.githubusercontent.com/65233860/212411049-3bbe700b-4c20-420c-a8d3-66341da66a5e.png)

| Your detail page | Actual design |
| --- | --- |
![Screenshot from 2023-01-13 20-14-13](https://user-images.githubusercontent.com/65233860/212411599-b6dea00e-45bb-44fc-bce5-25116670d28a.png)  |  ![Screenshot from 2023-01-13 20-13-38](https://user-images.githubusercontent.com/65233860/212411600-cf1521ce-e084-4588-9c6d-156450d8e01f.png)